tests/ut/ir/transforms/test_basic_memory_reuse.py

```
@@ -1013,5 +1013,111 @@ def main(
+                self,
+                input_a: pl.Tensor[[64, 64], pl.FP32],
+                input_b: pl.Tensor[[64, 64], pl.FP32],
+                output: pl.Tensor[[64, 64], pl.FP32],
+            ) -> pl.Tensor[[64, 64], pl.FP32]:
+                gate_init: pl.Tile[[64, 64], pl.FP32] = pl.load(input_a, [0, 0], [64, 64])
+                up_init: pl.Tile[[64, 64], pl.FP32] = pl.load(input_b, [0, 0], [64, 64])
+                for _i, (gate_acc, up_acc) in pl.range(4, init_values=(gate_init, up_init)):
+                    chunk: pl.Tile[[64, 64], pl.FP32] = pl.load(input_a, [0, 0], [64, 64])
+                    gate_new: pl.Tile[[64, 64], pl.FP32] = pl.add(gate_acc, chunk)
+                    up_new: pl.Tile[[64, 64], pl.FP32] = pl.add(up_acc, chunk)
+                    gate_out, up_out = pl.yield_(gate_new, up_new)
```

Ruff (0.15.6)

[warning] Unpacked variable `up_out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)